src/main/java/no/itera/bloggingplatform/repository/database/DbPostRepository.java

```diff
@@ -0,0 +1,142 @@
+package no.itera.bloggingplatform.repository.database;
+
+import no.itera.bloggingplatform.model.Author;
+import no.itera.bloggingplatform.model.Category;
+import no.itera.bloggingplatform.model.Post;
+import no.itera.bloggingplatform.repository.AuthorRepository;
+import no.itera.bloggingplatform.repository.CategoryRepository;
+import no.itera.bloggingplatform.repository.PostRepository;
+import org.springframework.context.annotation.Profile;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static no.itera.bloggingplatform.constant.ProfileDefinition.DATABASE_STORAGE;
+
+@Repository
+@Profile(DATABASE_STORAGE)
+public class DbPostRepository extends AbstractDbRepository implements PostRepository {
+
+    public DbPostRepository(JdbcTemplate jdbcTemplate) {
+        super(jdbcTemplate);
+    }
+
+    String tableName = "post";
```

Comment:
Inac ked definujes nieco ako nemenne, tak by to malo byt `private static final`. Plus naming convention je dat to upercase, t.z. `TABLE_NAME`